versionedRouter.js

```diff
@@ -95,34 +96,70 @@ const functionsEnabled = () => {
 };
 
 async function handleCdkV2(destName, parsedEvent, flowType) {
-  try {
-    const workflowEngine = await getWorkflowEngine(destName, flowType);
+  const workflowEngine = await getWorkflowEngine(destName, flowType);
 
-    const result = await workflowEngine.execute(parsedEvent);
-    // TODO: Handle remaining output scenarios
-    return result.output;
-  } catch (err) {
```

Comment:
The CDK errors have to be handled separately as we need to emphasize the priority of these errors (see TODOs). This is to configure sensitive alert thresholds for them.
Specifically, the scope must be set to CDK for workflow library errors (not execution).

Reply:
It may not be a priority for the comparison testing but leaving this comment here as a reminder to address it later.

Reply:
Added back stats without breaking the comparison.